### ISSUE TITLE
Broken link to travis job - includes extra quotes (")

### ISSUE THREAD

**Issue by dennisschagt** [lemurheavy/coveralls-public#1395]
The links to the relevant Travis Job appears broken.
It includes superfluous quotes around the Travis Job ID.

For an example, see https://coveralls.io/builds/28429766 which links to
`https://travis-ci.org/newsboat/newsboat/jobs/"643862704"` instead of
`https://travis-ci.org/newsboat/newsboat/jobs/643862704`

![image](https://user-images.githubusercontent.com/4629607/73497317-cff2df00-43ba-11ea-8375-ef231b664a93.png)

**Comment by Minoru** [comment]
I think I'm the one to blame for this: [my coverage submission script](https://github.com/newsboat/newsboat/blob/8997c0fc912f208e9daf31654d700384e1406fb1/submit-to-coveralls.sh#L30) puts some command-line arguments into a variable, and then [uses that variable as an argument](https://github.com/newsboat/newsboat/blob/8997c0fc912f208e9daf31654d700384e1406fb1/submit-to-coveralls.sh#L39). My intent was to avoid problems with spaces and such if they ever appear in TRAVIS_JOB_ID. However, in reality these escaped double quotes inside the variables are passed verbatim to the command—probably because the shell doesn't try to un-quote things after expanding variables.

In other words, my script unintentionally puts double quotes in JSON. Coveralls simply displays what it's been given by my script.

That's fixed in master now: https://github.com/newsboat/newsboat/commit/2dbe81d6122154d60f7a7c13964b24edb519d03f

This might just be that final little detail that I was missing to fix the reports in Newsboat. I'm waiting for builds to finish right now :)

**Comment by dennisschagt** [comment]
Nice, thanks for the quick fix!
